src/modules/widgets/index.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * Internal Dependencies
+ */
+import EventsList from '@moderntribe/events/widgets/events-list';
+import './style.pcss';
+
+const { registerBlockType } = wp.blocks;
+const { registerLegacyWidgetBlock } = wp.widgets;
+
+const blocks = [
+	EventsList,
+];
+
+blocks.forEach( block => {
+	const blockName = `tribe/${ block.id }`;
+	registerLegacyWidgetBlock();
```

Comment:
`registerLegacyWidgetBlock` does not need to be run for each block we register. This only needs to be run once above the `blocks` constant above.

```suggestion

wp.widgets.registerLegacyWidgetBlock();

const blocks = [
	EventsList,
];

blocks.forEach( block => {
	const blockName = `tribe/${ block.id }`;
```